account_security_extra/views/account_security_extra_menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.menu_action_account_form" model="ir.ui.menu">
+        <!-- Overriden: account.account_account_menu -->
```

Comment:
Could you make the same uninstall hook as #65 for this undo this change when addon is removed?